_pages/dev/cors.md

```diff
@@ -0,0 +1,164 @@
+# Cross-Origin Resource Sharing (CORS)
+
+While CORS is a standard mechanism on the web, it's considered rather complicated and blocks new Spartacus developer often during their first setup. This is why we spend an article on explaing _how_ CORS affects Spartacus, and _what_ is required to enable the right CORS setup.
+
+## What is CORS?
+
+CORS is a standard mechanism on the web to enable cross-domain requests from a web application to a server on a different domain. Browsers block Cross-Origin requests (or simply put: _cross-domain requests_), whenever the required HTTP headers are not available on the response.
+
+The response headers are dictated by the server, which is why the server must be setup to generate the right headers. In the SAP commerce backend, those headers are configurable in a generic fashion, using a CorsFilter. Project properties can be used to configure this for each node, or an installation script (ImpEx) can be used to install to each node. See for more information on the setup in the section below.
+
+## CORS Headers
+
+There are various CORS headers that come into play to specify whether the origin is allowed with or without certain methods, headers, cookies, etc. Each of the configurations and it's use in spartacus is discussed in the following sections.
+
+### allowedOrigins
+
+In development the allowed origins are often configured with a `*`, meaning every client is allowed. In production this should be more precise, to avoid abuse of the API.
```

Comment:
Perhaps indicate that this acts as a "whitelist" for requests from other domains?

Reply:
It'd also be good to have a link to CX docs around these

Reply:
done